apps/test-app/frontend/src/components/tree-widget/TreeWidget.tsx

```diff
@@ -273,3 +290,51 @@ function parseTreeNodeItem(node: HierarchyNode): DelayLoadedTreeNodeItem {
     autoExpand: node.autoExpand,
   } as DelayLoadedTreeNodeItem;
 }
+
+function createInfoNode(parentNode: TreeNodeItem | undefined, message: string, type?: InfoTreeNodeItemType): PresentationInfoTreeNodeItem {
+  const id = parentNode ? `${parentNode.id}/info-node` : `/info-node/${message}`;
+  return {
+    id,
+    parentId: parentNode?.id,
+    label: PropertyRecord.fromString(message),
+    message,
+    isSelectionDisabled: true,
+    children: undefined,
+    type: type ?? InfoTreeNodeItemType.Unset,
+  };
+}
+
+interface StatelessTreeNodeRendererProps extends TreeNodeRendererProps {
+  onLimitReset: () => void;
+}
+
+function StatelessTreeNodeRenderer(props: StatelessTreeNodeRendererProps) {
+  const nodeItem = props.node.item;
+
+  if (isPresentationInfoTreeNodeItem(nodeItem)) {
+    return (
+      <TreeNode
+        isLeaf={true}
+        label={
+          <span>
+            <Text isMuted>
+              <span>{nodeItem.message}</span>
+              {nodeItem.type === InfoTreeNodeItemType.ResultSetTooLarge && (
+                <span>
+                  <span> - </span>
+                  <UnderlinedButton onClick={() => props.onLimitReset()}>
+                    {` ${IModelApp.localization.getLocalizedString("Sample:controls.reset-hierarchy-level-limit")}.`}
```

Comment:
why is there a space before the localized string?

Reply:
Removed.